WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -129,6 +129,10 @@ - (void)viewDidLoad
     self.refreshControl = [[UIRefreshControl alloc] init];
     [self.refreshControl addTarget:self action:@selector(refreshTriggered:) forControlEvents:UIControlEventValueChanged];
 
+    if (self.isCancellable) {
```

Comment:
I'd consider removing the `isCancellable` property and using instead:

`if (self.navigationController.navigationBar.backItem == nil) {`

So that the "Done" button will always be shown when the back button is missing (ie: when this is shown modally).